### PR TITLE
Created Staff and Participant Dashboard

### PR DESCRIPTION
**Pull request by julianrkung**
Created Staff and Participant Dashboard. Styled according to Joelene's high-fi designs

Had to change some style/logic in CaseNote and Paperwork as well since they weren't responding well to changes in viewport.

### Current problem
Unloop logo does not respond well to changes in viewport. Don't know yet how to fix but trying to work with a column-reverse grid container.

### Related PRs
@kyuwonk @erinysong 
Might conflict with some of Fred's PR on Paperwork-related components and if Fred/Erin are working on CaseNote components I updated some style for that too so watch out for conflicts/breaks.

### Tests Performed, Edge Cases
To test you'll want to make sure to have participants with enough Paperworks/CaseNotes that they overflow to ensure that the container scroll work properly.

CC: @adowski

### CODE REVIEW

**Comment by kyuwonk** [comment]
You may have to cherrypick your commit onto the master branch to resolve conflicts with master right now

File: app/javascript/components/ParticipantShowPage/index.js [hunk 1]
```diff
@@ -43,16 +51,77 @@ class ParticipantShowPage extends React.Component {
...
+                  direction="column"
+                  spacing={10}
+                >
+                  {/* Grid items used to pad Unloop logo to bottom of screen */}
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
```
Comment (kyuwonk) [inline app/javascript/components/ParticipantShowPage/index.js]: I would instead set the Grid container that surrounds the entire navbar to be `justify="space-between"` and surround a Grid item around the actions and another around the logo
